packages/deployer/test/sample-project/test/UpgradeModule.test.js

```diff
@@ -80,4 +81,66 @@ describe('UpgradeModule', () => {
       assert.equal(await UpgradeModule.getImplementation(), routerAddress);
     });
   });
+
+  describe('when attempting to destroy the implementation with a malicious contract', () => {
```

Comment:
This new section in tests destroyed the implementation in previous commits, and now it just shows that upgrading the implementation of the current system implementation to the destroyer reverts.